tests/irdl/test_attribute_definition.py

```diff
@@ -693,14 +693,6 @@ def generic_constraint_coercion(args: tuple[Any]) -> AttrConstraint:
     def from_list(data: list[A]) -> ListData[A]:
         return ListData(data)
 
-    def verify(self) -> None:
-        for idx, val in enumerate(self.data):
-            if not isinstance(val, Attribute):
-                raise VerifyException(
-                    f"{self.name} data expects attribute list, but element "
-                    f"{idx} is of type {type(val)}."
-                )
-
```

Comment:
The type system already enforces this, surely we don't need to check this again? Pylance now complains